src/Components/PollItems.js

```diff
@@ -0,0 +1,106 @@
+import React, { useEffect, useState } from 'react'
+import { useSelector, useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { getUser } from '../redux/reducers/authSlice';
+import { MdDelete } from "react-icons/md";
+import { FaEdit } from "react-icons/fa";
+import { FaChartArea } from "react-icons/fa";
+
+
+import { ADMIN_ID } from '../utils/constantValue';
+
+const PollItems = ({ poll, increaseVoteCount,showDeleteModal,showPollChartModal}) => {
+    const [selectedOption, setSelectedOption] = useState(null);
+    const [voted, setVoted] = useState(false);
+
+    const { user } = useSelector((state) => state.auth);
+    const dispatch = useDispatch();
+
+    useEffect(() => {
```

Comment:
you should not be using getUser here because this will call every time when user come on this page. It should be set from login functions or app js page @vikash-kanauja